### PR TITLE
Fix MySQL collation parsing from DSN

### PR DESCRIPTION
**Pull request by stanislas-m**

### CODE REVIEW

**Comment by stanislas-m** [comment]
Thanks @mclark4386!

**Comment by u007** [comment]
@stanislas-m did you see my comment?

**Comment by stanislas-m** [comment]
@u007 On this PR? I don't remember something like that.

File: connection_details_test.go [hunk 1]
```diff
@@ -43,6 +43,26 @@ func Test_ConnectionDetails_Finalize_MySQL_DSN(t *testing.T) {
 	r.Equal("database", cd.Database)
 }
 
+func Test_ConnectionDetails_Finalize_MySQL_DSN_collation(t *testing.T) {
+	r := require.New(t)
+
+	url := "mysql://user:pass@(host:port)/database?collation=utf8mb4_general_ci"
```
[the remark this answers is not in the document]
Comment (stanislas-m) [reply connection_details_test.go]: It should work, but I'll add more tests to make sure. :)

**Pull request merged**